cypress/integration/learn/index.js

```diff
@@ -1,4 +1,4 @@
-/* global cy expect */
```

Comment:
Would love to improve cypress testing as well, but `cy` isn't an environment that ESLint supports. Here is the full list: https://eslint.org/docs/2.0.0/user-guide/configuring#specifying-environments

`eslint-plugin-cypress` can help resolve this though (another selling point 😄).